.github/workflows/cicd.yml

```diff
@@ -139,7 +138,7 @@ jobs:
             }
 
             log_note "run TDVM"
-            cp "$acon_root/aconcli/acon-startvm" . && ./aconcli run -n "$docker_id.json" -c :5532 || {
+            cp "$acon_root/scripts/acon-startvm" . && ATD_BIOS=OVMF.fd ATD_KERNEL=kernel.img ./aconcli run -n "$docker_id.json" -c :5532 || {
```

Comment:
Why not use `-f` flag to spe`cify the `acon-startvm` path? Alternatively, `aconcli` can run directly within `$acon_root/aconcli/` directory (without copying).